fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -5177,8 +5177,31 @@ public void replayModifyTableProperty(short opCode, ModifyTablePropertyOperation
     }
 
     // The caller need to hold the db write lock
-    public void modifyTableReplicationNum(Database db, OlapTable table, Map<String, String> properties) {
+    public void modifyTableReplicationNum(Database db, OlapTable table, Map<String, String> properties) throws DdlException {
         Preconditions.checkArgument(db.isWriteLockHeldByCurrentThread());
+        short newReplicationNum = (short) -1;
+        try {
+            newReplicationNum = Short.valueOf(properties.get(PropertyAnalyzer.PROPERTIES_REPLICATION_NUM));
```

Comment:
This should be done in `analyze()` of `ModifyTablePropertyClause.java`.
And you can call `PropertyAnalyzer.analyzeReplicationNum()`.

Reply:
ok.